app/plugins.js

```diff
@@ -307,6 +311,13 @@ function requirePlugins() {
 
       // populate the name for internal errors here
       mod._name = basename(path);
+      try {
+        // eslint-disable-next-line import/no-dynamic-require
+        mod._version = require(resolve(path, 'package.json')).version;
```

Comment:
this will be `sync` and block other stuff, better to do something like:
```js
const getVersion = (path) => new Promise((resolve, reject) => {
  fs.readFile(resolve(path, 'package.json'), (error, result) => {
    if (error) return reject(error)
    try {
      const { version } = JSON.parse(result.toString())
      return resolve(version)
    } catch (error) {
      return reject(error)
    }
  })
})

```
or just use https://github.com/sindresorhus/load-json-file 😄